www/js/controllers.js

```diff
@@ -231,12 +198,14 @@ angular.module('starter.controllers', ['urlConstant'])
   var submitParams = function() {
     $http.get(apiUrl+'search?duration='+$scope.searchParameters.duration+"&difficulty="+$scope.searchParameters.difficulty+"&scenery="+$scope.searchParameters.scenery+"&distance="+$scope.searchParameters.distance+"&hk="+$scope.searchParameters.regions[0].checked+"&kln="+$scope.searchParameters.regions[1].checked+"&nt="+$scope.searchParameters.regions[2].checked).success(function(data, status, xhr){
```

Comment:
can you create an url variable above the $http request, and concatenate over several lines, for more readability?